rpi-gpio.js

```diff
@@ -181,7 +181,14 @@ function Gpio() {
                 exportPin(pinForSetup, next);
             },
             function(next) {
-                setEdge(pinForSetup, edge, next);
+              async.retry({times: 10, interval: 100},
+                function(cb){
+                  setEdge(pinForSetup, edge, cb);
```

Comment:
Look at it this way, if the function can't write because of permissions, it doesn't _itself_ know that retry is the appropriate response. Maybe the permissions are just wrong.

OTOH, the control flow's order and timing are the cause of a potential race, so it is probably best to handle the potential race in the control flow code.